### PR TITLE
chore: mantine-fy use refresh server

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: #8295 

### Description:

Job details drawer

example

https://github.com/lightdash/lightdash/assets/7611706/bddfbae8-dca1-4692-b60c-b5e56e9d7f36

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/hooks/useRefreshServer.tsx [hunk 1]
```diff
@@ -13,42 +12,32 @@ import { lightdashApi } from '../api';
...
 import useToaster from './toaster/useToaster';
 
-export const jobStepStatusLabel = (
-    status: JobStepStatusType,
-): { label: string; icon: IconName } => {
+export const jobStepStatusLabel = (status: JobStepStatusType) => {
     switch (status) {
         case JobStepStatusType.DONE:
-            return { label: 'Completed', icon: 'tick-circle' };
+            return 'Completed';
```
Comment (joaoviana) [inline packages/frontend/src/hooks/useRefreshServer.tsx:18]: icon not being used

File: packages/frontend/src/hooks/useRefreshServer.tsx [hunk 2]
```diff
@@ -13,42 +12,32 @@ import { lightdashApi } from '../api';
...
-    status: JobStatusType,
-): { label: string; icon: IconName } => {
+export const jobStatusLabel = (status: JobStatusType) => {
     switch (status) {
         case JobStatusType.DONE:
-            return {
-                label: 'Successfully synced dbt project!',
-                icon: 'tick-circle',
-            };
+            return 'Successfully synced dbt project!';
```
Comment (joaoviana) [inline packages/frontend/src/hooks/useRefreshServer.tsx:34]: icon not being used

File: packages/frontend/src/components/JobDetailsDrawer/JobDetailsDrawer.styles.tsx [hunk 3]
```diff
@@ -1,98 +0,0 @@
...
-    path {
-        fill: ${({ status }) => statusInfo(status).color};
-    }
-`;
-
-export const ErrorMessageWrapper = styled.div`
-    margin-top: 10px;
-    word-wrap: break-word;
-    hyphens: auto;
-`;
```
Comment (IrakliJani) [inline packages/frontend/src/components/JobDetailsDrawer/JobDetailsDrawer.styles.tsx:98]: yes!

File: packages/frontend/src/components/JobDetailsDrawer/index.tsx [hunk 4]
```diff
@@ -103,57 +138,77 @@ const JobDetailsDrawer: FC = () => {
...
+                        p="sm"
+                        spacing="xs"
+                        sx={{
+                            borderRadius: 3,
+                        }}
+                    >
+                        <Box pt={2}>
+                            <StepIcon step={step} />
+                        </Box>
+                        <Stack spacing={1}>
```
Comment (IrakliJani) [inline packages/frontend/src/components/JobDetailsDrawer/index.tsx:184]: we have `one` and `two`

File: packages/frontend/src/providers/ActiveJobProvider.tsx [hunk 5]
```diff
@@ -40,7 +40,7 @@ export const ActiveJobProvider: FC = ({ children }) => {
     const toastJobStatus = useCallback(
         (job: Job | undefined) => {
             if (job && !isJobsDrawerOpen) {
-                const toastTitle = `${jobStatusLabel(job?.jobStatus).label}`;
+                const toastTitle = jobStatusLabel(job?.jobStatus);
```
Comment (IrakliJani) [inline packages/frontend/src/providers/ActiveJobProvider.tsx:43]: where did the label disappear from here?

**Review by IrakliJani — COMMENTED** [review]
reviewing pr env now

**Review by IrakliJani — APPROVED** [review]
@joaoviana good stuff!

im surprised you did not use this https://v6.mantine.dev/core/stepper/#vertical-orientation 😆 even has a loading state https://v6.mantine.dev/core/stepper/#loading-state

not a blocker!

but there's one comment about `label` which should be checked before merging

File: packages/frontend/src/providers/ActiveJobProvider.tsx (hunk 5 above)
Comment (joaoviana) [reply packages/frontend/src/providers/ActiveJobProvider.tsx:43]: sure! `jobStatusLabel` util was returning an object {icon:..., label:...} and `icon` wasn't being used anywhere! Now, `jobStatusLabel` returns the label `string`

**Comment by joaoviana** [comment]
> @joaoviana good stuff!
> 
> im surprised you did not use this https://v6.mantine.dev/core/stepper/#vertical-orientation 😆 even has a loading state https://v6.mantine.dev/core/stepper/#loading-state
> 
> not a blocker!
> 
> but there's one comment about `label` which should be checked before merging

I thought about that, and could have added it here, but just want to make sure we're not using blueprint, that's all 😅 ty though!

**Pull request merged**